app/src/ui/toolbar/push-pull-button.tsx

```diff
@@ -53,9 +53,11 @@ export class PushPullButton extends React.Component<IPushPullButtonProps, {}> {
 
     const progressValue = progress ? progress.value : undefined
 
+    // an unborn repository can be published, otherwise check
+    // for any sort of network activity
     const disabled = this.props.branchExists
       ? this.props.networkActionInProgress || !!this.props.progress
```

Comment:
Wouldn't these checks still apply to an unborn branch? I'm not sure how exactly, but it doesn't seem like we should ignore them?

Reply:
This is the only place that `branchExists` is used, so let's :fire: it to the ground